tic_tac_toe/game.py

```diff
@@ -9,9 +12,11 @@ def _position_is_empty_in_board(position, board):
 
     Returns True if given position is empty, False otherwise.
     """
-    pass
-
-
+    row = position[0]
+    column = position[1]
+    return board[row][column] == "-"
```

Comment:
In our solution we checked to see if the position was a O or X. We should have just done this instead.